assets/lib/nav.scss

```diff
@@ -0,0 +1,460 @@
+// Select the mobile menu style
+// styles include: sidebar, menu, none
+$nav-mobile-style: menu;
+
+// Responsive breakpoint
+$nav-min-width: $on-laptop;
+
+:root {
+  //// Defaults
+
+  // Padding defaults
+  --nav-padding: .75ex;
+  --nav-padding-ratio: 2.5;
+  --nav-padding-y: var(--nav-padding);
+  --nav-padding-x: calc(var(--nav-padding-y) * var(--nav-padding-ratio));
+
+  // Color defaults
+  --color-nav: transparent;
+  --color-nav-text: var(--color-text);
+  --color-nav-menu: var(--color-grey-light);
+  --color-nav-menu-top: var(--color-nav-menu);
+  --color-nav-menu-top-text: var(--color-nav-text);
+  --color-nav-highlight: var(--color-brand);
+  --color-nav-highlight-text: #fff;
+  --nav-active: transparent;
+
+  // Icons
+  --nav-icon-opacity: 0.5;
+  --nav-icon-menu: "≡";
+  --nav-icon-right: "⯈";
+  --nav-icon-down: "▾";
+
+  // Styling defaults (shadow and rounded radius)
+  --nav-menu-shadow: 2px 2px 2px rgba(var(--color-text), 0.125);
+  --nav-menu-radius: 3px;
+
+  // Animation timing defaults
+  --nav-animation: 150ms;
+  --nav-menu-animation: var(--nav-animation) * 2;
+  --nav-mobile-menu-animation: all 300ms ease-in-out;
+
+  // These should be set in theme.scss/theme.scss or minima already...
+  // but just in case, we're defining fallbacks here
+  --on-palm: 600px;
+  --on-laptop: 800px;
+
+  // Mobile menu colors
+  --nav-mobile-text: white;
+  --nav-mobile-text-shadow: 1px 1px 0 darken(var(--color-brand, ) 30%);
+  --nav-mobile-menu: darken(var(--color-brand, ) 20%);
+  --nav-mobile-menu-highlight: mix(var(--nav-mobile-menu, ) var(--nav-mobile-text, ) 80%);
+  --nav-mobile-button-size: 32px;
+  --nav-mobile-font-size: 16px;
+  --nav-mobile-item-height: 48px;
+  --nav-mobile-item-padding: 0 16px;
+  --nav-mobile-menu-padding: 8px 0;
+  --nav-mobile-menu-min-width: 56px * 3;
+
+  // Native iOS or Android fonts when installed, else use website fonts
+  --nav-mobile-font-family: -apple-system-font, "Roboto", "Droid Sans", var(--font-base-family);
+}
+
+////////////////////////////////////////////////////////////////////////
+
+@mixin no-select {
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
+
+.nav-check {
+  display: none;
+}
+
+@media screen and (min-width: #{$nav-min-width + 1}) {
+  nav.menu, %nav-menu {
+    position: relative;
+    z-index: 100;
+    cursor: default;
+
+    // Cleafix hack to make nav act more like block
```

Comment:
Clearfix?

Reply:
Ah, we can probably drop clearfix and do something more modern these days. A lot has changed since our site launched. :grin: